ledger/block/src/helpers/target.rs

```diff
@@ -18,13 +18,39 @@ use console::prelude::{Network, Result, ensure};
 /// A safety bound (sanity-check) for the coinbase reward.
 pub const MAX_COINBASE_REWARD: u64 = 190_258_739; // Coinbase reward at block 1.
 
-/// Calculate the block reward, given the total supply, block time, coinbase reward, and transaction fees.
+/// A the maximum block interval in seconds. This is used to upper bound the block interval in the V2 block reward calculation to
+/// prevent the block reward from becoming too large in the event of a long block interval.
+const V2_MAX_BLOCK_INTERVAL: i64 = 60; // 1 minute.
```

Comment:
Perhaps in the discussion of `ARC-0042` there should be some further discussion about how high this number *should* be. I think we're seeing in `monte-carlo` simulations run by @d0cd that emissions are lowered for shorter block times, but actually **increased** for higher block times (this makes sense with the math proposed). IMHO `60` is too high and increases emissions in the case that block times go over 10 seconds. 

This is likely to be the case as more validators and more traffic come onto the network, and perhaps antithetical to the original intent of the `ARC`. What do you think @raychu86 (and anyone else involved in this discussion) about proposing lowering this number to a maximum of perhaps `20` to incentivize the network to keep lower block times and not risk inflation in the case that block times go over 10 seconds?

Reply:
See comment https://github.com/AleoNet/snarkVM/pull/2569#issuecomment-2457458722 for my thoughts.